test/e2e/sync_plugin_test.go

```diff
@@ -257,6 +257,46 @@ func podTemplateTest(name string, ta *testArgs) {
 
 }
 
+func javaBuilderPodTemplateTest(name string, ta *testArgs) {
+	bc := &buildv1.BuildConfig{}
+	bc.Name = name
+	bc.Spec = buildv1.BuildConfigSpec{
```

Comment:
Are we doing any `gofmt` linting on this project? Two spaces instead of tabs seems way out of standard go format.

Reply:
yes this has come up before ... I'm running `gofmt` .... not sure what is going on here

I'll post a specific before / after example later on.  And if as goland upgrade as addressed it, then yes, we'll do it in a separate PR :-)

Reply:
with @adambkaplan 's awesome fine about .editorconfig the tabbing now looks normal in this PR ...check it out